awsproviderlint/passes/AWSAT003/README.md

```diff
@@ -0,0 +1,44 @@
+# AWSAT003
```

Comment:
If this check is going to be dual purpose (generally the previous checks have not been), it should show all examples  of flagged/passing code. e.g. using the `aws_region` data source.

Reply:
It will find regions, even if they are part of an AZ so perhaps the designation as dual purpose is not necessary.

Reply:
Very true! 👍